pkg/services/sqlstore/team_test.go

```diff
@@ -161,34 +161,34 @@ func TestIntegrationTeamCommandsAndQueries(t *testing.T) {
 				require.Equal(t, qAfterUpdate.Result[0].Permission, models.PERMISSION_ADMIN)
 			})
 
-			t.Run("Should default to member permission level when updating a user with invalid permission level", func(t *testing.T) {
-				sqlStore = InitTestDB(t)
-				setup()
-				userID := userIds[0]
-				team := team1
-				err = sqlStore.AddTeamMember(userID, testOrgID, team.Id, false, 0)
-				require.NoError(t, err)
-
-				qBeforeUpdate := &models.GetTeamMembersQuery{OrgId: testOrgID, TeamId: team.Id, SignedInUser: testUser}
-				err = sqlStore.GetTeamMembers(context.Background(), qBeforeUpdate)
-				require.NoError(t, err)
-				require.EqualValues(t, qBeforeUpdate.Result[0].Permission, 0)
-
-				invalidPermissionLevel := models.PERMISSION_EDIT
-				err = sqlStore.UpdateTeamMember(context.Background(), &models.UpdateTeamMemberCommand{
-					UserId:     userID,
-					OrgId:      testOrgID,
-					TeamId:     team.Id,
-					Permission: invalidPermissionLevel,
-				})
-
-				require.NoError(t, err)
-
-				qAfterUpdate := &models.GetTeamMembersQuery{OrgId: testOrgID, TeamId: team.Id, SignedInUser: testUser}
-				err = sqlStore.GetTeamMembers(context.Background(), qAfterUpdate)
-				require.NoError(t, err)
-				require.EqualValues(t, qAfterUpdate.Result[0].Permission, 0)
-			})
+			// t.Run("Should default to member permission level when updating a user with invalid permission level", func(t *testing.T) {
```

Comment:
Please don't comment out tests 🙂

Either use `t.Skipf("explain why the test is skipped")` or delete it if it's incorrect.

Reply:
@sakjur My bad! I missed the commented code and I meant to address it later. Let me fix this and I'll ping you when it is ready.

Thank you for pointing it out.

Reply:
@sakjur PTAL